src/rpc/handlers/AccountObjects.cpp

```diff
@@ -56,49 +56,50 @@ doAccountNFTs(Context const& context)
     if (auto const status = getLimit(request, limit); status)
         return status;
 
-    ripple::uint256 marker;
-    if (auto const status = getHexMarker(request, marker); status)
+    ripple::uint256 pageMarker;
+    ripple::uint256 tokenMarker;
```

Comment:
I'm not excited about the default values of these being used as flags that they are not present.  Consider using `std::optional<ripple::uint256>`.